packages/shared/src/components/widgets/common.ts

```diff
@@ -1,10 +1,9 @@
 import classed from '../../lib/classed';
 import { ElementPlaceholder } from '../ElementPlaceholder';
 
-export const WidgetContainer = classed(
-  'div',
-  'border border-theme-divider-tertiary rounded-16',
-);
+export const widgetClasses = 'border border-theme-divider-tertiary rounded-16';
```

Comment:
This should just be used on the WidgetContainer, elsewhere widget container should just be extended with whatever extra classes.